project2/styles.css

```diff
@@ -0,0 +1,238 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #508d8b;
+    color: black;
+    font-family: termina, sans-serif;
+}
+
+a {
+    text-decoration: none;
+}
+
+
+h2,
+h3,
+h4,
+p {
+    padding-left: 10px;
+}
+
+
+h2,
+h3,
+p {
+    padding-right: 10px;
+}
+
+
+h2 {
+    color: #ae274d;
+    font-family: termina, sans-serif;
+    font-size: 10.5px;
+    font-style: normal;
+    font-weight: 400;
+    letter-spacing: 3px;
+    text-transform: uppercase;
+}
+
+
+h3 {
+    color: #428784;
+    font-family: bebas-neue-by-fontfabric, sans-serif;
+    font-size: 34px;
+    font-style: normal;
+    font-weight: 700;
+    letter-spacing: 3px;
+    line-height: 1;
+    margin-top: 0;
+    margin-bottom: 0;
+}
+
+
+h4 {
+    color: #938f8f;
+    display: inline-block;
+    font-family: bebas-neue-by-fontfabric, sans-serif;
+    font-size: 10px;
+    letter-spacing: 2px;
+    line-height: 1;
+}
+
+h4+h4 {
+    border-left: 1px solid gray;
+    padding-right: 3px;
+}
+
+
+img {
+    width: 100%;
+}
+
+
+p {
+    font-family: montserrat, sans-serif;
+    font-size: 10.5px;
+    font-style: normal;
+    font-weight: 300;
+    margin-inline-end: 6px;
+}
+
+
+ul {
+    margin: 0;
+    padding-inline-start: 0;
```

Comment:
This property isn't in wide use yet:
https://caniuse.com/#feat=mdn-css_properties_padding-inline-start

You can instead use `padding-left: 0;` to undo the default browser list styles.